src/app/async/api.ts

```diff
@@ -0,0 +1,23 @@
+export const fetchData = async (): Promise<Article[]> => {
+    try {
+        const response = await fetch('https://jsonplaceholder.typicode.com/comments');
```

Comment:
ok is good but please don't use directly the URL you need to pass that as a parameter 
```
export const fetchData = async (URL: string): Promise<Article[]> => {
    try {
        const response = await fetch(URL);
        ...
```
and you can use it as follows
```
fetchData (`process.env.BASE_API_URL/${service}`)
```
this is only an example you need to made changes based in your requirements